Website/Packages/datax-pipeline/src/modules/flowDefinition/components/info/infoSettingsContent.jsx

```diff
@@ -73,6 +77,26 @@ export default class InfoSettingsContent extends React.Component {
             </div>
         );
     }
+
+    renderSaveButton() {
+        //Will be enabled when user has access 
```

Comment:
//Will be enabled when user has access and the token is not empty

Reply:
Updated comment